### PR TITLE
data: zaidahmad25

### PR DESCRIPTION
**Pull request by zaidahmad25**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4710"></a>

### CODE REVIEW

**Review by sidharthviz — APPROVED** [review]
Looks Good 🎉

**Review by Saadat-B — APPROVED** [review]
Looks Good 👍
Thank you for your contribution! 🎉

**Review by stevenovak123 — APPROVED** [review]

**Comment by zaidahmad25** [comment]
Thank you for approving guys! But I am not able to see my profile. It says "zaidahmad25" not found.
Also just for confirmation, ```https://linkfree.eddiehub.io/{username}``` should be the link right?

**Comment by Saadat-B** [comment]
The changes have only been approved by us. You'll be able to see your profile once the maintainers approve it and merge the pull request. Please be patient it will be merged soon 😄

**Comment by zaidahmad25** [comment]
Oh okay! I was not aware. Thank you for the reply :)

File: data/zaidahmad25.json
Comment (priyankarpal) [inline data/zaidahmad25.json:19]: Change the icon to `SiLinkedin`
``` suggestion
      "icon": "SiLinkedin"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/zaidahmad25.json
Comment (zaidahmad25) [reply data/zaidahmad25.json:19]: Hi there! I don't know how to change the contents of the file once a user makes a pull request. So please guide me for the same.

Comment (SaraJaoude) [reply data/zaidahmad25.json:19]: @priyankarpal the icon `FaLinkedinIn` is a valid icon (see screenshot) so this should not be a blocker for merging the PR. 

Screenshot 2023-02-12 at 09 48 16



> Hi there! I don't know how to change the contents of the file once a user makes a pull request. So please guide me for the same.

@zaidahmad25 you do not need to change the contents of your file if you want to keep this icon.  For future reference, here is a link to our Docs which guides you how to make changes (see the Open Pull Request section in answer to your question): https://linkfree.eddiehub.io/docs/how-to-guides/editing

Comment (zaidahmad25) [reply data/zaidahmad25.json:19]: Thank you!

**Pull request merged**
